SourceX/main.cpp

```diff
@@ -22,8 +22,20 @@ static std::string build_cmdline(int argc, char **argv)
 	return str;
 }
 
+void quit(){
+	static bool completed = false;
+	if(completed) return;
+	completed = true;
+	dvl::init_cleanup();
+	dvl::SaveGamma();
+	SDL_Quit();

```

Comment:
As mentioned I would prefer that you don't move the call to SDL_Quite, having it close to init means that it easier to maintain as it has a logical position.